kubernetes/assets/monitors/monitor_deployments_replicas.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "[kubernetes] Monitor Kubernetes Deployments Replica Pods",
 	"type": "query alert",
-	"query": "avg(last_15m):avg:kubernetes_state.deployment.replicas_desired{*} by {kube_namespace,kube_deployment} - avg:kubernetes_state.deployment.replicas_available{*} by {kube_namespace,kube_deployment} >= 2",
+	"query": "avg(last_15m):avg:kubernetes_state.deployment.replicas_desired{*} by {kube_cluster_name,kube_namespace,kube_deployment} - avg:kubernetes_state.deployment.replicas_available{*} by {kube_cluster_name,kube_namespace,kube_deployment} >= 2",
 	"message": "More than one Deployments Replica's pods are down in Deployment {{kube_namespace.name}}/{{kube_deployment.name}}.",
```

Comment:
we could add `kube_cluster_name` in the message as well ?

Reply:
@mfpierre said that kube_cluster_name is optional, so we should probably keep it out of the message or it would show "N/A"